### ISSUE TITLE
Allow non-plotted columns to be shown in tooltips

### ISSUE THREAD

**Issue by joeb0894** [metabase/metabase#30220]
**Is your feature request related to a problem? Please describe.**
In the previous version of metabase, when there was a series breakout and a "Stack" stacking in a bar chart, by passing the mouse on single part of the column, a black box appeared giving extra info that was inside the query. In the current version, the extra info has been substituted by a percentage on the total. Since the percentage info is also one of the other info I can insert directly in the query, giving that as a default and not changeble thing is difficult to manage.

**Describe the solution you'd like**
I would like to have the previous black box visualization, where there was the extra info present in the query.

**Describe alternatives you've considered**
By using "Don't stack" stacking, the black box is like the previous version but with the problem that, with a lot of data, the chart becomes incomprehensible.

**How important is this feature to you?**
It's a crucial part of my daily job. It helps me a lot by checking data on a single dashboard searching for outliers, strange behaviours and so on. By not having those data on the chart, the same job is very complicated and long.

**Additional context**
The first screenshot is about the current situation, the second is the "Don't stack" situation that has the same values in the black box that there were before also in the "Stack" situation
![Screenshot 2023-04-19 125400](https://user-images.githubusercontent.com/131256071/233056072-76404d1a-8e3b-4d5e-aa79-626f5afc2061.png)
![Screenshot 2023-04-19 130329](https://user-images.githubusercontent.com/131256071/233056098-09ce4f28-2236-4913-8d8c-0a8c53d06c90.png)

**Comment by paoliniluis** [comment]
maybe you're looking for https://github.com/metabase/metabase/issues/13096?

**Comment by joeb0894** [comment]
> maybe you're looking for #13096?

Not really, it was something already present in the previous metabase version. 
It's not about having data visible directly on charts (also because with very thin parts it will be confusing), but about having the appearing black box filled with those data.

**Comment by brunobergher** [comment]
@cdeweyx Have we handled this with the ECharts tooltip migration?

**Comment by cdeweyx** [comment]
> [@cdeweyx](https://github.com/cdeweyx) Have we handled this with the ECharts tooltip migration?

Yeah closing this
